src/Sylius/Behat/Context/Setup/ZoneContext.php

```diff
@@ -114,4 +145,77 @@ public function theStoreDoesNotHaveAnyZonesDefined()
             $this->zoneRepository->remove($zone);
         }
     }
+
+    /**
+     * @Given the store has a zone :zoneName with code :code
+     * @Given the store also has a zone :zoneName with code :code
+     */
+    public function theStoreHasAZoneWithCode($zoneName, $code)
+    {
+        $zone = $this->zoneFactory->createTyped(ZoneInterface::TYPE_ZONE);
+        $zone->setCode($code);
+        $zone->setName($zoneName);
+
+        $this->sharedStorage->set('zone', $zone);
+        $this->zoneRepository->add($zone);
+    }
+
+    /**
+     * @Given /^(it) has the ("([^"]*)" country) member$/
+     * @Given /^(this zone) has the ("([^"]*)" country) member$/
+     * @Given /^(it) also has the ("([^"]*)" country) member$/
```

Comment:
If we are using regex anyway, won't it be worthy to merge same step with optional `also`?